helm/hocs-templates/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
I don't think we need this file, because the packages are only going to be built in CI and so the `.gitignore` will be sufficient (and even then, developer-specific ignores like .bak and .vscode should live in a global gitignore file).

It certainly doesn't need to contain `.svn`(!)